### ISSUE TITLE
ssh key updating does not work

### ISSUE THREAD

**Issue by timogoebel** [kaorimatz/fog-scaleway#1]
First of all. Thanks for this gem. It's quite neat.
However, when I try to do the following,

```ruby
account = Fog::Account.new(:provider => :scaleway, :scaleway_token => 'abcdef')
user = account.users.first
user.ssh_public_keys << {'key' => 'ssh-rsa base64 my-super-name'}
user.save
```

I get this error:
```
Fog::Scaleway::Account::InvalidRequestError: Validation Error
	phone_number:
		- extra keys not allowed
```

I didn't change the phone number at all. I know, Scaleway's documentation is bad. My question is: Am I doing something wrong or do we need to handle this? If I'm not mistaken, Scaleway doesn't allow updating the phone number through the control panel. Might this be an issue?

I also can't understand these lines of code:

```ruby
            options[:ssh_public_keys] = ssh_public_keys.map do |k|
              { 'key' => k['key'] }
            end
```
options[:ssh_public_keys] is already an array of hashes with the key `key`, so why is the map needed?

Thanks you for your help. I'm happy to assist where I can.

**Comment by kaorimatz** [comment]
Thank you for taking your time to report this issue. As you said, fog-scaleway gem should not send phone_number as it is not allowed to change it with API. I'll fix it.

> options[:ssh_public_keys] is already an array of hashes with the key key, so why is the map needed?

It strips all keys besides `key` because `options[:ssh_public_keys]` returned from `GET /users/{user_id}` contains `fingerprint` key which causes "extra keys not allowed" error if we don't remove it from the request body for `PATCH /users/{user_id}`.

**Comment by kaorimatz** [comment]
@timogoebel I just fixed and released v0.2.1. Please let me know If you still have problems. Thank you for your report :)

**Comment by timogoebel** [comment]
@kaorimatz : Thanks you for fixing this so quickly. I just tested this and everything works now.
